mathjax3-ts/adaptors/lite/Parser.ts

```diff
@@ -339,6 +339,10 @@ export class LiteParser implements MinDOMParser<LiteDocument> {
      * @return{string}              The serialized element (like innerHTML)
      */
     public serializeInner(adaptor: LiteAdaptor, node: LiteElement) {
+        const PCDATA = (this.constructor as typeof LiteParser).PCDATA;
```

Comment:
The `liteAdaptor` is pretty minimal, so it may well need to be subclassed.  This allows the subclass's `PCDATA` to be used.  Without this, you would have to override `serializeInner()` in the subclass  just to change the explicit `LiteParser` reference.